tools/generators/epic-generator/index.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { addProjectConfiguration, ProjectType, serializeJson, stripIndents } from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { execSync } from 'child_process';
+import epicGenerator from './index';
+import { readFileSync } from 'fs';
+
+jest.mock('child_process');
+// @ts-ignore
+const execSyncMock = <jest.Mock<string>>execSync;
```

Comment:
- lets not use ts-ignore when it;s not needed.
- lets not use assertion via this way. it cannot be used consistently and confuses generic type instantiation. we should always use `as` if we need to assert type value in this case it looks like following:

```ts
const execSyncMock = (execSync as unknown) as jest.Mock<string>;
const readFileSyncMock = (readFileSync as unknown) as jest.Mock<string>;
```